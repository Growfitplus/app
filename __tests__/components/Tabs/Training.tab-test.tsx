import TrainingTab from '@/components/Tabs/Training.tab';
import { render } from '@testing-library/react-native';

describe('Training Tab', () => {
  it('should render without crashing', () => {
    const { toJSON } = render(<TrainingTab focused />);

    expect(toJSON()).toMatchSnapshot();
  });

  it('should not add bg when not focused', () => {
    const { toJSON } = render(<TrainingTab focused={false} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
