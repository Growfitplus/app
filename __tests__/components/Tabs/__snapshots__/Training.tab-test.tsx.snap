// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Training Tab should not add bg when not focused 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "none",
      "borderRadius": 14,
      "height": 40,
      "justifyContent": "center",
      "width": 40,
    }
  }
>
  <RNSVGSvgView
    bbHeight={24}
    bbWidth={24}
    fill="none"
    focusable={false}
    height={24}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        {
          "flex": 0,
          "height": 24,
          "width": 24,
        },
      ]
    }
    width={24}
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M9 4.75h6A4.25 4.25 0 0 1 19.25 9v6A4.25 4.25 0 0 1 15 19.25H9A4.25 4.25 0 0 1 4.75 15V9A4.25 4.25 0 0 1 9 4.75ZM3 9a6 6 0 0 1 6-6h6a6 6 0 0 1 6 6v6a6 6 0 0 1-6 6H9a6 6 0 0 1-6-6V9Zm9.875-1h-1.75v5.375H16v-1.75h-3.125V8Z"
        fill={
          {
            "payload": 4280494648,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;

exports[`Training Tab should render without crashing 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#00FFC2",
      "borderRadius": 14,
      "height": 40,
      "justifyContent": "center",
      "width": 40,
    }
  }
>
  <RNSVGSvgView
    bbHeight={24}
    bbWidth={24}
    fill="none"
    focusable={false}
    height={24}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        {
          "flex": 0,
          "height": 24,
          "width": 24,
        },
      ]
    }
    width={24}
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M9 4.75h6A4.25 4.25 0 0 1 19.25 9v6A4.25 4.25 0 0 1 15 19.25H9A4.25 4.25 0 0 1 4.75 15V9A4.25 4.25 0 0 1 9 4.75ZM3 9a6 6 0 0 1 6-6h6a6 6 0 0 1 6 6v6a6 6 0 0 1-6 6H9a6 6 0 0 1-6-6V9Zm9.875-1h-1.75v5.375H16v-1.75h-3.125V8Z"
        fill={
          {
            "payload": 4280494648,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
